flink-connectors/flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/PulsarSource.java

```diff
@@ -97,16 +94,14 @@
      * PulsarSourceBuilder}.
      */
     PulsarSource(
-            Configuration configuration,
+            SourceConfiguration sourceConfiguration,
```

Comment:
For source related changes, I guess the changes to the source is related to sink because we want to align the naming conventions and hierarchy , but maybe consider putting in a separate commit ?

Reply:
Yep. Since the code is still in progress. We can separate the commits after everything is done.